test/e2e/backup_test.go

```diff
@@ -105,9 +120,9 @@ var _ = Describe("[Restic] Velero tests on cluster using the plugin provider for
 				backupName = fmt.Sprintf("backup-%s-%s", bsl, uuidgen)
 				restoreName = fmt.Sprintf("restore-%s-%s", bsl, uuidgen)
 
-				Expect(RunKibishiiTests(client, cloudProvider, veleroCLI, veleroNamespace, backupName, restoreName, bsl)).To(Succeed(),
+				Expect(RunKibishiiTests(client, cloudProvider, veleroCLI, veleroNamespace, backupName, restoreName, bsl, true)).To(Succeed(),
```

Comment:
This test should also use the `useVolumeSnapshots` value.

Reply:
Thanks, done